Sources/BraintreeThreeDSecure/V2UICustomization/BTThreeDSecureV2ButtonCustomization.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+import CardinalMobile
+
+/// Button customization options for 3D Secure 2 flows.
+@objcMembers public class BTThreeDSecureV2ButtonCustomization: BTThreeDSecureV2BaseCustomization {
+
+    // MARK: - Public Properties
+
+    /// Color code in Hex format. For example, the color code can be “#999999”.
+    public var backgroundColor: String? {
+        get { _backgroundColor }
+        @objc(setBackgroundColor:) set {
+            _backgroundColor = newValue
+            (cardinalValue as? ButtonCustomization)?.backgroundColor = newValue
+        }
+    }
```

Comment:
This pattern exists in every class that was converted. Currently these properties are accessed like this in each language:
```
// Swift
let labelCustomization = BTThreeDSecureV2LabelCustomization()
labelCustomization.headingTextColor = "#A020F0"

// Objective-C
BTThreeDSecureV2LabelCustomization *labelCustomization = [BTThreeDSecureV2LabelCustomization new];
[labelCustomization setHeadingTextColor:@"#A020F0"];
```
Since there are going to be other breaking changes in the conversion of 3D Secure, it did not seem compelling to also break this interface. From the merchant point of view if they are using any of these customization classes, they will have the exact same experience pre and post the conversion outlined here. 

That said if someone feels incredibly strongly about this, we can certainly discuss!

Reply:
Thanks for the explanation here - down to minimize breaking changes at this point, and agree to keep!